var/spack/repos/builtin/packages/ncbi-rmblastn/package.py

```diff
@@ -12,11 +12,18 @@ class NcbiRmblastn(AutotoolsPackage):
     homepage = "http://www.repeatmasker.org/RMBlast.html"
     url      = "ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.9.0/ncbi-blast-2.9.0+-src.tar.gz"
 
+    version('2.11.0', sha256='d88e1858ae7ce553545a795a2120e657a799a6d334f2a07ef0330cc3e74e1954')
     version('2.9.0', sha256='a390cc2d7a09422759fc178db84de9def822cbe485916bbb2ec0d215dacdc257')
+
+    patch('isb-2.11.0+-rmblast.patch', when="@2.11.0")
```

Comment:
Can you add a comment above the patch to state its provenance and explain why it is needed?

Reply:
Yes, I can add a comment. I will add here as well. Briefly, the patch is provided by the project.
```
To compile from source:
Download NCBI Blast+ and rmblast patch file:
ncbi-blast-2.11.0+-src.tar.gz
isb-2.11.0+-rmblast.patch.gz
----
http://www.repeatmasker.org/RMBlast.html
```

I followed what was done for the previous version but was wondering if there was a way for spack to work with a gzipped patch like that.